x/superfluid/keeper/migrate_test.go

```diff
@@ -452,21 +401,18 @@ func (suite *KeeperTestSuite) TestMigrateSuperfluidUnbondingBalancerToConcentrat
 			suite.Require().NoError(err)
 
 			// Run slashing logic and check if the new and old locks are slashed.
-			suite.SlashAndValidateResult(ctx, newGammLockId, concentratedLockId, poolIdEntering, tc.percentOfSharesToMigrate, valAddr, *balancerLock, true)
+			suite.SlashAndValidateResult(ctx, newGammLockId, concentratedLockId, clPoolId, tc.percentOfSharesToMigrate, valAddr, *balancerLock, true)
```

Comment:
ditto here and below, do you think it's possible to add more checks and validations for partial unbonding logic?

Reply:
since #5162  also touches same logic, will request this test case if needed over there